platform-ui-skin/src/main/webapp/skin/less/social/portlets/profileSettings/Style.less

```diff
@@ -0,0 +1,104 @@
+/*
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2022 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+@import "../../../variables.less";
+@import "../../../mixins.less";
+#profileSettings {
+    background: @baseBackground!important;
+    #settingsHeader {
+        background: @baseBackground;
+       #filterSettingsSelect {
+          height: 40px;
+          width: 150px;
+        }
+    }
+
+    #profileSettingFormDrawer{
+        .selectLanguageFilter {
+          height: 40px!important;
+          width: -moz-fit-content!important;
+          width: fit-content!important;
+        }
+        .removeIcon{
+          color: @errorColor;
+        }
+        .addLabelBtn{
+          color: @primaryColor;
+  		  font-size: 16px;
+ 		  cursor: pointer;
+ 		  .addLabelIcon{
+            color: @primaryColor;
```

Comment:
not necessary, use `primary--text` class
Besides, on `v-icon` you can use `color="primary"`

```suggestion
```